javascript/javascript3/week3/promises/promises.js

```diff
@@ -0,0 +1,29 @@
+const promise1 = fetch('https://api.github.com/search/repositories?q=user:JyothiNandyala');
+const promise2 = fetch('https://api.github.com/search/repositories?q=user:ccshah298');
+const promise3 = fetch('https://api.github.com/search/repositories?q=user:Enqira');
+
+const repo = document.querySelector('#repo');
+
+Promise.all([promise1, promise2, promise3])
+    .then(responses => {
+        const json = responses.map(response => response.json());
+        return Promise.all(json)
```

Comment:
not sure if this Promise.all is necessary. But if it works as you want it, ignore my comment.